### ISSUE TITLE
Accessibility Issues from user test

### ISSUE THREAD

**Issue by robguthrie** [loomio/loomio#4417]
-    [ ] no main region on the marketing homepage
-    [ ] when no activity (ie: in last 6 weeks) in dashboard give a message
-    [ ] focus is not (and should be) moving to h1 on navigation.
-    [ ] H1 in context not working?
-    [ ] thread context needs to be first in DOM order
-    [ ] post button should be next in tab order after adding a comment

**Comment by juliencarnot** [comment]
Last item looks like a duplicate of #4324, and I don't get what the problem is (see comment)...
